### ISSUE TITLE
Incorrect import of @types/socket.io

### ISSUE THREAD

**Issue by aljazerzen** [nestjs/nest#512]
## I'm submitting a bug report
<pre><code>
[ ] Regression 
[x] Bug report
[ ] Feature request
[ ] Documentation issue or request
[ ] Support request => Please do not submit support request here, instead post your question on Stack Overflow.
</code></pre>

## Current behavior
`/@nestjs/websockets/adapters/io-adapter.d.ts` is not importing socket.io correctly. It is a bus with new version of @types/socket.io package

## Environment

<pre><code>
Nest version: 4.6.6
Socket.io: 2.0.4
@types/socket.io: 1.4.32
</code></pre>

**Comment by EreckGordon** [comment]
I too am getting a compilation error,
```
node_modules/@nestjs/websockets/adapters/io-adapter.d.ts(14,27): error TS2503: Cannot find namespace 'SocketIO'.
node_modules/@nestjs/websockets/adapters/io-adapter.d.ts(16,35): error TS2503: Cannot find namespace 'SocketIO'.
```
if i swap out /// <reference types="socket.io" /> for import * as SocketIO from 'socket.io' it compiles, but changing node_modules is a very ephemeral fix, so hopefully i'm just missing something else

**Comment by BorntraegerMarc** [comment]
I think it is not a NestJS error. See here: https://github.com/DefinitelyTyped/DefinitelyTyped/issues/24330

**Comment by EreckGordon** [comment]
`yarn add @types/socket.io@1.4.31` fixed the problem because 1.4.32 typings are broken.

**Comment by mmenik** [comment]
@nestjs/websocket: 5.0.1
@types/socket.io: 1.4.36

don't fix
